libs/cli/langchain_cli/integration_template/docs/vectorstores.ipynb

```diff
@@ -17,35 +17,88 @@
    "source": [
     "# __ModuleName__VectorStore\n",
     "\n",
-    "This notebook covers how to get started with the __ModuleName__ vector store.\n",
+    "This notebook covers how to get started with the __ModuleName__ vector store."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36fdc060",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "- TODO: Update with relevant info.\n",
+    "- TODO: Update minimum version to be correct.\n",
+    "\n",
+    "To access __ModuleName__ vector stores you'll need to create a/an __ModuleName__ account, get an API key, and install the `__package_name__` integration package."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "64e28aa6",
+   "metadata": {
+    "vscode": {
+     "languageId": "raw"
+    }
+   },
+   "source": [
+    "%pip install -qU \"__package_name__>=MINIMUM_VERSION\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9695dee7",
+   "metadata": {},
+   "source": [
+    "### Credentials\n",
+    "\n",
+    "- TODO: Update with relevant info.\n",
     "\n",
-    "## Installation"
+    "Head to (TODO: link) to sign up to __ModuleName__ and generate an API key. Once you've done this set the __MODULE_NAME___API_KEY environment variable:"
```

Comment:
Is the logic across API_KEYs conditional? 
We have plenty of vectorstores without API_KEYs.

Reply:
For the integrations that don't need one, they can remove this line!

This is just a template for bootstrapping docs with the CLI